cpp/src/arrow/ipc/util.h

```diff
@@ -19,6 +19,8 @@
 
 #include <cstdint>
 
+#include "arrow/type.h"
```

Comment:
Hmm... why?

Reply:
This was an artifact of an intermediate state of the patch, will remove